setup.py

```diff
@@ -71,7 +71,8 @@ def _get_constant(name):
         entry_points={'gui_scripts':
                       ['qutebrowser = qutebrowser.qutebrowser:main']},
         zip_safe=True,
-        install_requires=['pypeg2', 'jinja2', 'pygments', 'PyYAML', 'attrs'],
+        install_requires=['pypeg2', 'jinja2', 'pygments', 'PyYAML', 'attrs',
+                          'importlib_resources>=1.1.0; python_version < "3.9"'],
```

Comment:
Please also add this to:

- The README
- `misc/requirements/requirements-qutebrowser.txt-raw`
- `_module_versions` in `qutebrowser/utils/version.py`
- and adjust `tests/unit/utils/test_version.py` accordingly (I can take care of that if you prefer)

Reply:
I'm guessing in requirements-qutebrowser.txt-raw it is unconditional?